ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-arm.yaml

```diff
@@ -27,6 +27,77 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: perfscale-aws-control-plane-120nodes-periodic
+  cron: 0 14 * * 4
+  steps:
+    cluster_profile: aws-perfscale
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "120"
+      OCP_ARCH: arm64
+      SIZE_VARIANT: large
```

Comment:
I know x86 calls out this way to set the control-plane node sizes... I would like for us to statically assign the instance type moving forward.

Reply:
Thanks! done. pls check new commit.